src/components/stats/h3-queries-stats.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import useSWR from 'swr';
+import { getH3QueriesStats } from '../../services/stats-api';
+import { Bar } from 'react-chartjs-2';
+import { theme } from '../../styles/global';
+
+interface H3Data {
+    is_valid: number | null;
+    amount: number;
+    hours: number;
+}
+
+const options = {
+    scales: {
+        yAxes: [
+            {
+                stacked: false,
+                scaleLabel: {
+                    display: true,
+                    labelString: 'hrs',
+                },
+            },
+        ],
+    },
+    legend: { display: false },
+};
+
+const H3QueriesStats: React.FunctionComponent = () => {
+    const { data, error } = useSWR('queries', getH3QueriesStats);
+
+    const generateChart = () => {
+        const barChartData = prepareBarChart(data as H3Data[]);
+        return <Bar data={barChartData} options={options} />;
+    };
+
+    const prepareBarChart = (indata: H3Data[]) => {
+        const hours: number[] = [];
+        const labels: string[] = [];
+        indata.forEach((e) => {
+            switch (e.is_valid) {
+                case 1:
+                    labels.push('Valid (hrs)');
+                    hours.push(e.hours);
+                    break;
+                case 2:
+                    labels.push('Invalid (hrs)');
```

Comment:
This label doesn't show up on the queries graph

Reply:
Also the colors seems a bit wonky for the queries graph. "unverified" is green and "valid" is red

Reply:
switched to a simpler implementation that actually works